core/domain/stats_jobs_one_off.py

```diff
@@ -1240,8 +1240,8 @@ def reduce(key, stringified_values):
                 yield (
                     'Item ID: %s, last updated: %s, state name: %s, '
                     'exp id: %s, error: %s' % (
-                        item_id, last_updated, state_name, exploration_id,
-                        error))
+                        item_id, last_updated, state_name.encode('utf-8'),
```

Comment:
I don't think so. We only need to encode it when we are displaying it. The state_name that's being logged to the storage model is the same that is logged to the 'states' property of an Exploration storage model.